h2/src/main/org/h2/command/dml/Select.java

```diff
@@ -82,6 +82,7 @@
     private boolean sortUsingIndex;
     private SortOrder sort;
     private int currentGroupRowId;
+    private boolean isExistsSubquery;
```

Comment:
Sure: https://github.com/h2database/h2database/pull/291/commits/cc37af347148b6d40901d8e36bbc9fecc922455f

That would make it the first documented field in this class, though ;)